lib/modules/hosts/derpibooru.js

```diff
@@ -0,0 +1,216 @@
+addLibrary('mediaHosts', 'derpibooru', {
+	name: 'Derpibooru',
+	domains: [
+		'derpiboo.ru',
+		'www.derpiboo.ru',
+		'derpibooru.org',
+		'www.derpibooru.org',
+		'trixiebooru.org',
+		'www.trixiebooru.org'
+	],
+	pathMatcher: /^\/(?:images\/)?(\d+)$/i,
+
+	detect: function(href, _elem) {
+		var mod = modules['showImages'].siteModules['derpibooru'];
+		href = new URL(href);
+
+		return mod.domains.indexOf(href.host) != -1 && mod.pathMatcher.test(href.pathname);
```

Comment:
Checking the domain in `detect()` is redundant (I know, it's in every single module, even `example.js`...I'll fix that too), and you can use `elem.pathname` without having to create a URL object, since it's an `<a>` element.